molecule/default/latest-converge.yml

```diff
@@ -10,6 +10,14 @@
         cache_valid_time: 600
       when: ansible_os_family == 'Debian'
       changed_when: false
+  vars:
+    nextcloud_apps:
+      calendar:
+      contacts:
+        source: "https://github.com/nextcloud-releases/contacts/releases/download/v5.0.2/contacts-v5.0.2.t>"
```

Comment:
There is a typo in link: `>"` shouldn't it be `tar.gz`?

Reply:
```suggestion
        source: "https://github.com/nextcloud-releases/contacts/releases/download/v5.0.3/contacts-v5.0.3.tar.gz"
```

Reply:
Thanks Wiktor & sorry for this undiscovered copy &  paste error. Actually v5.0.3 does not exist and so I have left it at 5.0.2.